etc/config-template.toml

```diff
@@ -64,6 +64,16 @@
 ## If not set, `addr` will be used.
 # advertise-addr = ""
 
+## If HTTP service is enabled. If the value is true, the Prometheus can pull the metrics directly 
+## through the HTTP address. Otherwise, the Prometheus will get the metrics through the PushGateway.
+# http-enabled = false
+
+## HTTP listening address.
+# http-addr = "127.0.0.1:9520"
```

Comment:
if we use K8s or docker, do we need to add an `advertise-http-addr` for outside use? @tennix

Reply:
No, only an HTTP server is OK.

Reply:
Bad name. It's not a general service, should mentioned metrics in the entry name.